en/services/camera_def.md

```diff
@@ -113,16 +115,33 @@ Parameter | Description
 -- | --
 CAM_APERTURE | Aperture
 CAM_EV | Exposure Compensation (usually only used for automatic exposure modes)
+CAM_VIDEV | Exposure compensation (Video)
+CAM_PHOTOEV | Exposure compensation (Photo)
 CAM_EXPMODE | Exposure Mode (Manual, Auto, Program Auto, Aperture Priority, etc.)
 CAM_ISO | ISO
+CAM_VIDISO | ISO (Video)
+CAM_PHOTOISO | ISO (Photo)
 CAM_METERING | Metering Mode
 CAM_SHUTTERSPD | Shutter speed
-CAM_VIDRES | Video Resolution (for video capture)
+CAM_VID_SHUTSPD | Shutter speed (Video)
```

Comment:
This is the only parameter that has underscore after VID. CAM_VIDSHUTSPD would be more consistent. Or is there a reason for it?